### PR TITLE
Shush compiler warnings

### PR DESCRIPTION
**Pull request by ItzSwirlz**
Tested on actual hardware.

If you want I can try and recreate the commit without the whitespaces if it bothers you (Sometimes nano will remove it too)

### CODE REVIEW

**Comment by icarter09** [comment]
Yes to removing the whitespaces and recommit. Thanks.

**Comment by ItzSwirlz** [comment]
Will do tomorrow, thanks for honest input

**Comment by ItzSwirlz** [comment]
There you go. I don't trust myself with a lot of pointer stuff yet, or division or that stuff. I'll leave it to the experts.

File: plugins/automount/csd-autorun.c [hunk 1]
```diff
@@ -987,5 +987,5 @@ csd_autorun_for_content_type (GMount               *mount,
                               CsdAutorunOpenWindow *callback,
```
Comment (mtwebster) [inline plugins/automount/csd-autorun.c:987]: this shouldn't be a pointer, it should be:
`CsdAutorunOpenWindow callback`

File: plugins/automount/csd-autorun.c [hunk 2]
```diff
@@ -987,5 +987,5 @@ csd_autorun_for_content_type (GMount               *mount,
...
                               gpointer              user_data)
 {
-    do_autorun_for_content_type (mount, content_type, callback, user_data);
```
Comment (mtwebster) [inline plugins/automount/csd-autorun.c:990]: This is correct as is.

File: plugins/automount/test-automount-dialog.c [hunk 3]
```diff
@@ -67,7 +66,7 @@ main (int argc, char *argv[])
 
             csd_autorun_for_content_type (mount,
                                           argv[1],
-                                          (CsdAutorunOpenWindow) autorun_show_window,
```
Comment (mtwebster) [inline plugins/automount/test-automount-dialog.c:70]: This is correct as is.

**Comment by ItzSwirlz** [comment]
should be fixed

**Pull request merged**